### DISCUSSION TITLE
family history questionnaire variable label descriptions

### DISCUSSION THREAD

**Discussion by burdinskid13** [now-i-know-my-abcd/docs#89] [Where in the World is…? (Help)]
I am not sure how to interpret some of the variable labels in the family history questionnaire. I included a screenshot of an example. is there another resource that gives more elaborate descriptions? Thanks so much!

![IMG_8796](https://github.com/user-attachments/assets/6afcff8a-07c9-4929-88bd-ca65f1a605c1)

**Comment by LucyWhitmore** [comment]
Hi! I believe these are the different response options that stem from the fam_history_5_yes_no question, which is: "5. Has ANY blood relative of your child ever had any problems due to drugs, such as: Marital separation or divorce; Laid off or fired from work; Arrests or DUIs; Drugs harmed their health; In a drug treatment program; Suspended or expelled from school 2 or more times; Isolated self from family, caused arguments or were high a lot. ¿ALGÚN pariente consanguíneo de su niño(a) ha tenido alguna vez problemas debido a las drogas, tal como una separación o divorcio, despido o cese laboral, arrestos o multas por conducir bajo la influencia de drogas, Ingresó a un programa de tratamiento contra la drogadicción, suspensión o expulsión de la escuela en 2 o más ocasiones, se aisló de la familia, provocó conflictos o con frecuencia estaba drogado?". 

So, all of those variables are combinations of relatives and response options. I believe the same structure is true for the other family history questions as well, and you can find the text of those questions by searching fam_history_5_yes_no, fam_history_6_yes_no, fam_history_7_yes_no, etc in the data dictionary (I'm not seeing questions 1-4, but those might be labeled differently, I'm not completely familiar with this measure). I hope that helps!

**Reply by burdinskid13** [reply]
Thanks so much! I really appreciate you getting back to me. What you described is in line with what I have been thinking too. However, other pieces of the family history questionnaire remain unresolved for me (eg. there is a question if a family member has no problems, but the answer to that doesn't seem to line up with another question about whether the family member has any problem. It isn't obvious to me how to clear up these mismatches. 

To get more clarity on this questionnaire, I would love to have a protocol in addition to the data dictionary. Are you aware of a way to get a written protocol that shows the complete question asked and the sequence of questions with codings for the variables?
